app/models/user.rb

```diff
@@ -62,6 +62,10 @@ def messages_in_organization(organization = Organization.current)
     messages.where('assignments.organization': organization)
   end
 
+  def notifications_in_organization(organization = Organization.current)
+    notifications.where(organization: organization)
+  end
```

Comment:
Can't `organic_on` be used as discussions do?

Reply:
you are absolutely right!